### ISSUE TITLE
abc

### ISSUE THREAD

**Issue by aglie** [k-eks/ThePictureBookOfFourierTransforms#14]
![letter a](https://user-images.githubusercontent.com/2978725/47724430-f88ba680-dc56-11e8-8367-c5e050bb2962.png)
![ft a](https://user-images.githubusercontent.com/2978725/47724432-faee0080-dc56-11e8-80bc-53a83d6e9335.png)

**Comment by aglie** [comment]
![letter b](https://user-images.githubusercontent.com/2978725/47724450-02ada500-dc57-11e8-884d-a7456cbc6e54.png)
![ft b](https://user-images.githubusercontent.com/2978725/47724454-05a89580-dc57-11e8-9ee6-51f6246a1c47.png)

**Comment by aglie** [comment]
![letter c](https://user-images.githubusercontent.com/2978725/47724466-0d683a00-dc57-11e8-87ec-8cd2cd46c80a.png)
![ft c](https://user-images.githubusercontent.com/2978725/47724478-0fca9400-dc57-11e8-8c04-c6c397505a79.png)

**Comment by aglie** [comment]
![letter g](https://user-images.githubusercontent.com/2978725/47724498-16f1a200-dc57-11e8-9d0f-4c3925c84feb.png)
![ft g](https://user-images.githubusercontent.com/2978725/47724504-19ec9280-dc57-11e8-8770-2f81ae56f40b.png)

**Comment by aglie** [comment]
Hi all, I made a tiny list of Fourier transforms of all the letters and some symbols, and it turned out much better than I thought. You can take a [look at the whole alphabet](https://arkadiysimonov.github.io/fourier/generate_abc.html), also, I have made a little [game ](https://arkadiysimonov.github.io/fourier/abc_game.html) you can try to suess the correct letters.

**Comment by k-eks** [comment]
This has definitely a high coolness factor! And that is an oddly satisfying game :)

Is there any chance to get a complete read-out of the images so that I can provide something for the wiki?

**Comment by aglie** [comment]
Sure. I will provide a 'save all' button.

вт, 30 окт. 2018 г. в 16:11, Gregor Hofer <notifications@github.com>:

**Comment by aglie** [comment]
Hi Gregor,

Here are all of the images. I also added the 'save' button to the abc generation script just in case.

[FT_abc.zip](https://github.com/k-eks/ThePictureBookOfFourierTransforms/files/2549475/FT_abc.zip)

**Comment by k-eks** [comment]
Dumping the packs..

![abc_atof_ft](https://user-images.githubusercontent.com/12133169/48270533-9e4eca80-e43a-11e8-8a12-dd79af97efef.png)
![abc_atof_im](https://user-images.githubusercontent.com/12133169/48270536-9e4eca80-e43a-11e8-88da-1a3a94eb27b4.png)
![abc_atoz_ft](https://user-images.githubusercontent.com/12133169/48270537-9e4eca80-e43a-11e8-88cc-ee735e7ee798.png)
![abc_atoz_im](https://user-images.githubusercontent.com/12133169/48270538-9ee76100-e43a-11e8-973e-b1d41954b490.png)
![abc_gtom_ft](https://user-images.githubusercontent.com/12133169/48270539-9ee76100-e43a-11e8-9291-19235d389a6c.png)
![abc_gtom_im](https://user-images.githubusercontent.com/12133169/48270540-9ee76100-e43a-11e8-932d-ddd37095682c.png)
![abc_ntot_ft](https://user-images.githubusercontent.com/12133169/48270542-9ee76100-e43a-11e8-9205-38564985b3f7.png)
![abc_ntot_im](https://user-images.githubusercontent.com/12133169/48270543-9f7ff780-e43a-11e8-939e-c23a9eec667c.png)
![abc_numbers_all](https://user-images.githubusercontent.com/12133169/48270545-9f7ff780-e43a-11e8-8d99-36bb84b74e9f.png)
![abc_utoz_ft](https://user-images.githubusercontent.com/12133169/48270546-9f7ff780-e43a-11e8-9fb0-1c34846d9021.png)
![abc_utoz_im](https://user-images.githubusercontent.com/12133169/48270547-9f7ff780-e43a-11e8-8442-aa11e94a6e7f.png)

**Comment by k-eks** [comment]
Oh and what is the name of the font?

**Comment by aglie** [comment]
Oh yes, you are correct, that is the Times New Roman. Should I change it to
Helvectica?


пт, 9 нояб. 2018 г. в 16:16, Gregor Hofer <notifications@github.com>:

**Comment by k-eks** [comment]
I don't think that it is necessary to change to font but it is important to know which font was used in case someone wants to redo these transforms for themselves and needs a comparison, and for that, the font ins important.

**Comment by aglie** [comment]
Oh, joke did not work. I mean you are in Switzerland, so obviously should
write in all small letters exclusively in helvetica ). HBB does by the way
;)

вт, 13 нояб. 2018 г. в 10:48, Gregor Hofer <notifications@github.com>:
